guides/v2.1/cloud/deploy/optimize-cloud-deploy.md

```diff
@@ -0,0 +1,19 @@
+---
+group: cloud-guide
+title: Optimize cloud deployment
+functional_areas:
+  - Cloud
+  - Deploy
+  - SCD
+---
+
+Site performance can suffer during the deployment process. The length of time a site is in maintenance mode when deploying to a production site depends on many factors, such as environment configuration and the amount of content a site contains. The first best-practice for optimizing your Cloud deployment is to [upgrade to use `{{site.data.var.ct}}`]({{page.baseurl}}/cloud/project/ece-tools-upgrade-project.html) to benefit from the package features, such as commands to create a backup of the database and verify environment configuration.
```

Comment:
I don't think "best practices" requires hyphenation.

Reply:
Yes, I don't even know why I did that. LOL